database/migrations/2021_03_11_003713_rename_printer_columns.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class RenamePrinterColumns extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('printer_supplies', function (Blueprint $table) {
```

Comment:
You can do these all in the same Schema::table block

Reply:
Nope, sqlite doesnt support it.